packages/engine/src/scene/components/IFrameComponent.ts

```diff
@@ -35,24 +35,23 @@ import { S } from '@ir-engine/ecs/src/schemas/JSONSchemas'
 import { addError, clearErrors } from '../functions/ErrorFunctions'
 
 const interactMessage = 'Click'
-const liquidCodeCallbackName = 'liquidCodeCallback'
+const iframeCallbackName = 'iframeCallback'
 
-const toggleOpen = (liquidCodeEntity: Entity) => {
-  const liquidCodeComponent = getComponent(liquidCodeEntity, LiquidCodeComponent)
-  setComponent(liquidCodeEntity, LiquidCodeComponent, { isOpen: !liquidCodeComponent.isOpen })
+const toggleOpen = (iframeEntity: Entity) => {
+  const iframeComponent = getComponent(iframeEntity, IFrameComponent)
+  setComponent(iframeEntity, IFrameComponent, { isOpen: !iframeComponent.isOpen })
 }
 
-export const LiquidCodeComponent = defineComponent({
-  name: 'LiquidCodeComponent',
-  jsonID: 'ir_liquid_code',
+export const IFrameComponent = defineComponent({
+  name: 'IFrameComponent',
+  jsonID: 'ir_iframe',
```

Comment:
json ID should follow the GLTF spec for extensions, and have the namespace be capitalized like `IR_iframe`